pkg/mount/dag_steps.go

```diff
@@ -608,3 +636,56 @@ func (s *State) RunKcryptUpgrade(g *herd.Graph, opts ...herd.OpOption) error {
 		return internalUtils.UpgradeKcryptPartitions()
 	}))...)
 }
+
+type LsblkOutput struct {
+	Blockdevices []struct {
+		Name     string      `json:"name,omitempty"`
+		Parttype interface{} `json:"parttype,omitempty"`
+		Children []struct {
+			Name     string `json:"name,omitempty"`
+			Parttype string `json:"parttype,omitempty"`
+		} `json:"children,omitempty"`
+	} `json:"blockdevices,omitempty"`
+}
+
+// MountESPPartition tries to mount the ESP into /efi
+// Doesnt matter if it fails, its just for niceness
+func (s *State) MountESPPartition(g *herd.Graph, opts ...herd.OpOption) error {
+	return g.Add("mount-esp", append(opts, herd.WithCallback(func(ctx context.Context) error {
+		if internalUtils.CheckEfiPartUUID() != nil {
+			internalUtils.Log.Debug().Msg("Not mounting Esp Partition as we think we are booting from removable media")
```

Comment:
```suggestion
			internalUtils.Log.Debug().Msg("Not mounting ESP Partition as we think we are booting from removable media")
```

Reply:
ACKCHUALLY it should be ESP without the partition. I hate the ESP name.